pkgs/shells/fish/plugins/hydro.nix

```diff
@@ -2,13 +2,13 @@
 
 buildFishPlugin rec {
   pname = "hydro";
-  version = "unstable-2022-02-21";
+  version = "unstable-2024-03-24";
 
   src = fetchFromGitHub {
     owner = "jorgebucaran";
     repo = "hydro";
-    rev = "d4875065ceea226f58ead97dd9b2417937344d6e";
-    sha256 = "sha256-nXeDnqqOuZyrqGTPEQtYlFvrFvy1bZVMF4CA37b0lsE=";
+    rev = "bc31a5ebc687afbfb13f599c9d1cc105040437e1";
+    sha256 = "sha256-0MMiM0NRbjZPJLAMDXb+Frgm+du80XpAviPqkwoHjDA=";
```

Comment:
Since this is using the SRI hash, please change it to `hash = "sha256-whatever";`